3-projects/README.md

```diff
@@ -0,0 +1,44 @@
+# 3-projects
+
+The purpose of this step is to setup folder structure, project, DNS and subnets used for applications.
```

Comment:
Would it make sense to also be able to add a list of project owners which are then assigned as owners? Or do you intend to keep any kind of ownership out of this CFT? 

A new project creation flow would then be:
1. existing engineer creates PR to the org tf foundation project
2. on merge, that engineer (or the project lead or the team if group) gets assigned owner privileges
3. all project-internal tf code now gets handled outside of this repo

Another consideration could be to be able to pass in the git repo URL of the project terraform code AND/OR to create the git repo through this project, then return a standardized GIT repo URL as an ouput .

https://www.terraform.io/docs/providers/google/r/sourcerepo_repository.html

Reply:
Hi @pascalwhoop - thanks for taking the time to review and your interest! Typically I would not recommend adding individual users as project owners or for other IAM permissions, with a preference for using groups. We could look at exposing the following variables from the project factory module `group_name` and `group_role` for the single project & standard projects module if that would be helpful? It might be best also to also move these feature requests to issues, so we can track & triage them?

Regarding the second request, we are evaluating adding an example which would setup a dedicated cloud build pipeline and source repo, which you could add some thoughts here:

https://github.com/terraform-google-modules/terraform-example-foundation/issues/10